.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+defaults: &defaults
+  docker:
+    - image: canonicalwebteam/dev
+  working_directory: /srv
+
+version: 2
+jobs:
+  test-site:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: yarn && pip3 install -r requirements.txt
+      - run:
+          name: Build the site resources
+          command: yarn run build
+      - run:
+          name: Run the site server
+          command: ./entrypoint 0.0.0.0:80
+          background: true
+      - run:
+          name: Check site is accessible
+          command: sleep 3 && curl --head --fail --retry-delay 5 --retry 10  --retry-connrefused http://localhost
+  lint-scss:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install yarn dependencies
+          command: yarn install
+      - run:
+          name: Lint scss with sass-lint
+          command: yarn run lint-scss
+  lint-python:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: pip3 install -r requirements.txt
+      - run:
+          name: Lint webapp with Black
```

Comment:
and flake8

Reply:
Thanks. Actually, this circleCI task which runs `yarn run lint-python` doesn't know and shouldn't care that "black" or "flake8" are used. So I changed the wording to "lint Python files"

Reply:
I prefer too!